trajectory_generator:
  rovers: ['RR04', 'RR06']
  dt: .02
  num_timesteps: 100
  goal_states: 
    RR06: [3.0, 3.0, 0.0, 0.0]
    RR04: [3.0, -3.0, 0.0, 0.78]
  x_bounds: [[-10.0, 10], [-10.0, 10.0], [-0.75, 0.75], [-inf, inf]]
  u_bounds: [[-1.0, 1.0], [-0.785, 0.785]]
  min_allowable_dist: 1.5